### PR TITLE
fix(admin): Re-adds rendering for comments 

### PR DESCRIPTION
**Pull request by jhechtf**
## Screen Shots

**BEFORE**

ss-before

**AFTER**
ss-after

### CODE REVIEW

File: apps/admin/package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "@repo/auth": "workspace:*",
     "@repo/db": "workspace:*",
     "@repo/emails": "workspace:*",
-    "@repo/lint": "workspace:*",
```
Comment (bautistaaa) [inline apps/admin/package.json:22]: we dont need these?

Comment (jhechtf) [reply apps/admin/package.json:22]: Dog I don't know im just trying to get this to pass and I'd allow satan himself access to my code editor for it.

Comment (jhechtf) [reply apps/admin/package.json:22]: OK IDK why this is still showing up, it's still in there so i think that's a bug?

**Comment by jhechtf** [comment]
@bautistaaa seems like everything short of the builds is passing now for reasons i don't know.

**Comment by jhechtf** [comment]
screw it im gonna close this and re-do the work

**Pull request closed without merging**